### ISSUE TITLE
Allow PTR in "forward" zones to support DNS-SD (RFC 6763)

### ISSUE THREAD

**Issue by he32** [msimerson/NicTool#250]
### NicTool version

2.35

### perl version

5.24.1

### OS version

Debian 9

### Expected behavior

In order to allow the manual registration of records required for
DNS-SD, as specified in RFC 6763, NicTool needs to allow the
registration of PTR records in "forward" zones.

It appears that this requires a change in the SQL database used
to run NicTool, like so:

update resource_record_type set forward=1 where name='PTR';

### Observed behavior

PTR record does not occur in drop-down menu of record types for
"forward" zones.

### Steps to reproduce

Edit a "forward" zone, see PTR record missing from drop-down
menu.

**Comment by msimerson** [comment]
That is correct. Those who need PTR records in forward zones can enable it as you discovered.
